.travis.yml

```diff
@@ -3,6 +3,7 @@ language: cpp
 sudo: false
 
 stages:
+  - cmake
```

Comment:
Should this be a stage after `checks` or maybe part of it or part of the `tests` stage?

Reply:
To me it makes sense to make it a part of the `tests` stage.

Reply:
I'm fine with either. Should I move it to tests?

Reply:
> Should I move it to `tests`?

Yes